scripts/e2e_live.sh

```
@@ -40,6 +40,25 @@ require_tools() {
   command -v node >/dev/null 2>&1 || { echo "[e2e][fatal] node is required" | tee -a "$LOG" >&2; exit 1; }
```

Good shift from defaults to explicit fixtures and fail-fast. A couple of small nits for require_auth():

- The Doppler context check currently looks for DOPPLER_PROJECT|CONFIG|ENVIRONMENT via `env | grep`. That can give false positives and is a subshell; consider checking variables directly in POSIX shell:

  if [ -z "${DOPPLER_PROJECT:-}" ] && [ -z "${DOPPLER_CONFIG:-}" ] && [ -z "${DOPPLER_ENVIRONMENT:-}" ]; then
    echo "[e2e][fatal] Missing Doppler context…" >&2; exit 1
  fi

- Validate required tools earlier: because doppler is assumed, require_tools could also check `command -v doppler` and provide a dedicated error if absent.

- Optional: also validate E2E_WORKFLOW_FILE exists up front when mutations are enabled, to fail early if the workflow path is misconfigured.